deploy/deploy-operator-go/main.go

```diff
@@ -0,0 +1,306 @@
+package main
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	appsv1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/apps/v1"
+	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/core/v1"
+	metav1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/meta/v1"
+	rbacv1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/rbac/v1"
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/yaml"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		// Download file
+		filePath, cleanup, err := downloadFile("https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/v1.2.0/deploy/crds/pulumi.com_stacks.yaml")
+		if err != nil {
+			return err
+		}
+		crds, err := yaml.NewConfigFile(ctx, "crds", &yaml.ConfigFileArgs{
+			File: filePath,
```

Comment:
You can supply a URL directly to the File argument. Was that not working for some reason?

Reply:
I forgot to put a link to it here it seems but I also ran into https://github.com/pulumi/pulumi-kubernetes/issues/1569. Looks. like remote config files are not handled in the Go implementation.

Reply:
Oh, hmm. Didn't realize that.